hashedindex/textparser.py

```diff
@@ -16,6 +16,12 @@ def __repr__(self):
         return '<NullStemmer>'
 
 
+class InvalidStemmerException(Exception):
+
+    def __init__(self):
+        super().__init__('Stemmer must be an object with a stem(str) function')
```

Comment:
I would prefer if the constructor was left unmodified and the details were just passed in :)